superbench/common/utils/device_manager.py

```diff
@@ -129,7 +129,7 @@ def get_device_row_remapped_info(self, idx):
         Return:
             remapped_metrics (dict): the row remapped information, None means failed to get the data.
         """
-        output = process.run_command('nvidia-smi -i {} -q'.format(idx))
+        output = process.run_command('nvidia-smi -i {} -q'.format(idx), quite=True)
```

Comment:
Why change this?

Reply:
because I think monitor's stdout(regular run nvidia-smi) is not nessessary? it looks very messy and hard to read

Reply:
Make sense.